### PR TITLE
SoftwareFreedomConservancy.QEMUGuestAgent version 102.7.0

### PR DESCRIPTION
**Pull request by kevinoid**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/37032)

### CODE REVIEW

File: manifests/s/SoftwareFreedomConservancy/QEMU/GuestAgent/102.7.0/SoftwareFreedomConservancy.QEMU.GuestAgent.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SoftwareFreedomConservancy.QEMU.GuestAgent
+PackageVersion: 102.7.0
+PackageLocale: en-US
+Publisher: RedHat
+PublisherUrl: https://github.com/virtio-win/virtio-win-pkg-scripts/blob/master/README.md
+PublisherSupportUrl: https://www.qemu.org/support/
+PackageName: QEMU Guest Agent
```
Comment (ItzLevvie) [inline manifests/s/SoftwareFreedomConservancy/QEMU/GuestAgent/102.7.0/SoftwareFreedomConservancy.QEMU.GuestAgent.locale.en-US.yaml:10]:
```suggestion
PackageName: QEMU guest agent
```

Comment (ItzLevvie) [reply manifests/s/SoftwareFreedomConservancy/QEMU/GuestAgent/102.7.0/SoftwareFreedomConservancy.QEMU.GuestAgent.locale.en-US.yaml:10]: This must match Control Panel for consistency.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Comment by jamespik** [comment]
Why is this app added as a nested folder of the QEMU app? By policy, this should be a side by side application under the SoftwareFreedomConservancy publisher?

**Comment by kevinoid** [comment]
Thanks @jamespik.  I must have overlooked that policy.  Where is it documented?  Is that the case even though it is part of the same project/source tree?

**Comment by kevinoid** [comment]
Updated PackageIdentifier to SoftwareFreedomConservancy.QEMUGuestAgent.  I still have the [same reservations about using "SoftwareFreedomConservancy."](https://github.com/microsoft/winget-pkgs/pull/37032#issuecomment-986940656) but am using it for consistency.

After looking at other packages, it's clear that third-level naming is typically used for editions/variants rather than components (although I don't see any documentation for these conventions).  Sorry for my earlier confusion.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**